Editor/Completion/XmlSchemaCompletionProvider.cs

```diff
@@ -136,7 +136,9 @@ public static string GetUri (string fileName)
 
 		public void EnsureLoaded ()
 		{
+#pragma warning disable VSTHRD002 // Avoid problematic synchronous waits
```

Comment:
should probably use IWaitIndicator

Reply:
IWaitIndicator is in Microsoft.VisualStudio.Text.Internal.dll, not sure we want to add this dependency?

Reply:
Looks like there are alternatives? https://github.com/dotnet/roslyn/commit/a5019f17d35a8750bfc4f47f826f7d607e128c11